music_player/setup.py

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env python
+
+from distutils.core import setup
+from catkin_pkg.python_setup import generate_distutils_setup
+
+d = generate_distutils_setup(
+   #  don't do this unless you want a globally visible script
+   # scripts=['bin/myscript'], 
```

Comment:
the server node `aaf_music_player_server.py` should be installed here! (and renamed to `music_player_server.py`)